src/scripting_api/event.js

```diff
@@ -170,6 +171,17 @@ class EventDispatcher {
       if (event.willCommit) {
         this.runValidation(source, event);
       } else {
+        if (source.obj._isChoice) {
+          const exportValue = savedChange.changeEx;
+          source.obj.value = exportValue;
```

Comment:
Is the `exportValue` variable used for anything, or was this mostly for documentation purposes? If so, maybe just:
```suggestion
          source.obj.value = savedChange.changeEx; // exportValue
```